docs/modules/plugins/pages/plugin-azure-functions.adoc

```diff
@@ -0,0 +1,67 @@
+= Azure functions Plugin
+
+The plugin provides functionality to interact with https://azure.microsoft.com/en-us/services/functions/.
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-azure-functions', version: '{current-version}')
+----
+
+== Configuration
+
+=== Authentication
+
+By default, plugin's authentication depends on correct configure of following environment variables.
+
+
+. `AZURE_CLIENT_ID` - for Azure client ID
+. `AZURE_CLIENT_SECRET` - for client secret
+. `AZURE_CLIENT_CERTIFICATE_PATH` - for client certificate
+. `AZURE_TENANT_ID` - for Azure tenant ID
+. `AZURE_SUBSCRIPTION_ID` - Azure subscription ID
+
+
+[TIP]
+
+One should set either `AZURE_CLIENT_SECRET` or `AZURE_CLIENT_CERTIFICATE_PATH`
+
+
+See the official https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity["Azure identity"] guide to get more details.
```

Comment:
`https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity#environment-variables["Azure identity"]`